doc/man3/EVP_set_default_properties.pod

```diff
@@ -27,10 +30,22 @@ given with I<libctx> (NULL signifies the default library context).
 Any previous default property for the specified library context will
 be dropped.
 
+EVP_default_properties_enable_fips() sets the 'fips=yes' to be a default property
+if I<enable> is non zero, otherwise it clears 'fips=yes' as a default property
+for the given I<libctx>. It merges the fips default property setting with any
+existing query strings that have been set via EVP_set_default_properties().
+
+EVP_default_properties_is_fips_enabled() indicates if 'fips=yes' is a default
+property for the given I<libctx>.
```

Comment:
`s/for/query for/`

It isn't doing this, it also accepts `?fips=no`.

Reply:
Its good enough..